examples/tracing/bitehist.py

```diff
@@ -16,22 +16,31 @@
 from __future__ import print_function
 from bcc import BPF
 from time import sleep
+import os
+
+# check "blk_account_io_completion" against /proc/kallsyms
+string = os.popen('cat /proc/kallsyms | ' + 
+			'grep "blk_account_io_completion"'
+			).read()
+if len(string.strip()) > 0: 
+    event = "blk_account_io_completion"
+else: 
+    event = "blk_account_io_done"
 
 # load BPF program
 b = BPF(text="""
 #include <uapi/linux/ptrace.h>
 #include <linux/blkdev.h>
-
 BPF_HISTOGRAM(dist);
 BPF_HISTOGRAM(dist_linear);
-
-int kprobe__blk_account_io_completion(struct pt_regs *ctx, struct request *req)
+int kprobe_blk_account_io_completion(struct pt_regs *ctx, struct request *req)
```

Comment:
I think using the kprobe_ prefix here is slightly misleading - it looks really similar to the kprobe__ automatic attach thing